servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -4417,9 +4417,10 @@ void RendererSceneRenderRD::_update_volumetric_fog(RID p_render_buffers, RID p_e
 
 		uint32_t cluster_screen_width = (rb->width - 1) / cluster_size + 1;
 		uint32_t cluster_screen_height = (rb->height - 1) / cluster_size + 1;
-		params.cluster_type_size = cluster_screen_width * cluster_screen_height * (32 + 32);
-		params.cluster_width = cluster_screen_width;
+
```

Comment:
This whitespace can be deleted, just to keep things clean

Reply:
Ok, done.